api/agent/stats.go

```diff
@@ -160,7 +168,9 @@ func (s *stats) Stats() Stats {
 const (
 	queuedMetricName    = "queued"
 	callsMetricName     = "calls"
-	runningSuffix       = "running"
+	runningMetricName   = "running"
 	completedMetricName = "completed"
 	failedMetricName    = "failed"
+	timedoutMetricName  = "timedout"
```

Comment:
I chose "timedout" because "timed out" is adjectival and describes the state of a function call, just like "completed", "failed",  "queued" and "running". You could say that the call was queued, the call was running, the call was timed out. That leaves "calls" (plural noun) as the odd one out. 

If it was not "timedout" I would suggest "timeouts" (plural noun), which would be consistent with "calls".

I'll leave this for now but don't have a strong view. Hopefully people won't need to type any of these very often anyway (which is just as well since a lot of metric names, such as the duration histograms, are much longer than this)

Reply:
on tense alone that makes sense, and certainly we could stand to improve our English. I guess these are somewhat well hidden and not very properly used, but the states used throughout fn tend to (I think 100%?) follow https://github.com/fnproject/fn/blob/master/api/agent/drivers/driver.go#L66 -- we have not done a very good job of using constants or ensuring that only these states exist wrt calls/containers. agree it's not a very big deal, just like to have everything uniform for uniformity's sake I guess.

Reply:
I've logged an issue https://github.com/fnproject/fn/issues/691 so we can continue this discussion there.